subsys/net/ip/Kconfig.mgmt

```diff
@@ -38,6 +38,15 @@ config NET_MGMT_EVENT_QUEUE_SIZE
 	  notification. Thus the size of this queue has to be tweaked depending
 	  on the load of the system, planned for the usage.
 
+config NET_MGMT_EVENT_QUEUE_TIMEOUT
+	int "Timeout for event queue"
+	default 10
+	range 1 10000
+	help
+	  Timeout in milliseconds for the event queue. If the queue is full,
+	  the oldest event will be removed and the new one will be added.
+	  This timeout is used to wait for the queue to be available.
```

Comment:
Using "popped" instead of "removed" might be more appropriate when referring to the oldest event.